src/Altinn.App.Core/Implementation/AppResourcesSI.cs

```diff
@@ -256,14 +256,14 @@ public string GetModelJsonSchema(string modelId)
         public LayoutSettings GetLayoutSettings()
         {
             string filename = Path.Join(_settings.AppBasePath, _settings.UiFolder, _settings.FormLayoutSettingsFileName);
-            string filedata = null;
             if (File.Exists(filename))
             {
-                filedata = File.ReadAllText(filename, Encoding.UTF8);
+                var filedata = File.ReadAllText(filename, Encoding.UTF8);
+                LayoutSettings layoutSettings = JsonConvert.DeserializeObject<LayoutSettings>(filedata)!;
+                return layoutSettings;
             }
 
-            LayoutSettings layoutSettings = JsonConvert.DeserializeObject<LayoutSettings>(filedata);
-            return layoutSettings;
+            throw new Exception($"Could not find file in {filename}");
```

Comment:
```suggestion
            throw new FileNotFoundException($"Could not find layoutsettings file: {filename}");
```
Throw a more acurate exception